components/layout_2020/lists.rs

```diff
@@ -16,24 +15,25 @@ use crate::replaced::ReplacedContents;
 pub(crate) fn make_marker<'dom, Node>(
     context: &LayoutContext,
     info: &NodeAndStyleInfo<Node>,
-) -> Option<Vec<PseudoElementContentItem>>
+) -> Option<(NodeAndStyleInfo<Node>, Vec<PseudoElementContentItem>)>
 where
     Node: NodeExt<'dom>,
 {
-    let style = info.style.get_list();
-    let node = match info.node {
-        Some(node) => node,
-        None => {
-            warn!("Tried to make a marker for an anonymous node!");
-            return None;
-        },
-    };
+    // TODO: use `PseudoElement::Marker` when we add it.
+    let marker_info = info.pseudo(
+        context,
+        style::selector_parser::PseudoElement::ServoLegacyText,
+    )?;
+    let style = marker_info.style.clone();
+    let list_style = style.get_list();
```

Comment:
Do we really need to clone the entire set of style values here? Seems like the rest of the  function could simply use a reference to the list style struct. Or if that's not true we could clone just the list style struct, instead of everything.

Reply:
Yep, we can use a reference here, I think. That said, a `clone()` here just clones the `Arc` and not the entire `ComputedValues`.